### PR TITLE
New `Platform` section in navbar

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
> **Warning**: Merge After https://github.com/gitpod-io/website/pull/3814

## Description

- New navbar column for "Platform" contains:
    - Cloud
    - Dedicated 

- Removed "Enterprise" column

## Related Issue(s)

Fixes [MKWEB-70 : update navbar with "platform" drop down](https://linear.app/gitpod/issue/MKWEB-70/update-navbar-with-platform-drop-down)

## How to test

- Open preview and see navbar

## Release QA Checklist ✔️

<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

- [ ] Test all links to make sure they are working correctly
- [x] Verify that all forms (sales, support, docs feedback)  are functioning properly and data is being captured correctly
- [x] Check all buttons and calls-to-action to ensure they are leading to the correct pages
- [x] Ensure that the website is mobile responsive and functions appropriately on different devices
- [x] Please make sure that page is best for Light/Dark Mode. ⇒ Text/Media Changes

## [Priority: High] Content Testing

- [x] Check all website copies for accuracy and spelling/grammar errors
- [x] Verify that all images and videos are displaying properly
- [x] Ensure that all downloadable files are working correctly
- [x] Check that all external links are functioning properly

## [Priority: Medium] Performance Testing

- [x] Try to use optimal media assets. ([Decision Record - How To](https://github.com/gitpod-io/website/blob/main/docs/adr/0003-optimal-assets-format.md))
- [x] Test website load times to ensure that the website is loading quickly
- [x] Verify that the website can handle high traffic volumes without crashing
- [x] Check that all website functionality is working properly under heavy load
- [x] Perform cross-browser testing to ensure that the website works properly on all major browsers (Google Chrome, Firefox, Edge)

## [Priority: Medium] Accessibility Testing

- [x] Ensure that the website meets basic accessibility guidelines.
- [x] Test that users with disabilities can easily navigate and use the website
- [x] Verify that all website content can be accessed using assistive technologies

By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.

- [x] Verify that the website is HTTPS secure and that there are no security vulnerabilities
- [x] Ensure that all sensitive data is being stored and transmitted securely

  </details>

### CODE REVIEW

File: src/lib/contents/dropdown.ts [hunk 1]
```diff
@@ -66,6 +66,23 @@ export const docsLinks: {
...
   },
 ];
+export const platformLinks: {
+  text: string;
+  href: string;
+  description: string;
+}[] = [
+  {
+    text: "Cloud",
+    href: "#",
```
Comment (Siddhant-K-code) [inline src/lib/contents/dropdown.ts:76]:
```suggestion
    href: "/cloud",
```

Accept this suggestion & merge PR after the merge of #3814

**Review by chrifro — APPROVED** [review]
Could you use the same style e.g. width as for the other drop-downs?
|how it looks|how it should be|
|-|-|
|Bildschirmfoto 2023-07-04 um 15 17 46|Bildschirmfoto 2023-07-04 um 15 18 25
- the link to /cloud doesn't work yet
- when opening the mobile drop-down, "cloud" is already selected? both options should have the same default state
Bildschirmfoto 2023-07-04 um 15 20 12

**Comment by Siddhant-K-code** [comment]
> Could you use the same style e.g. width as for the other drop-downs?

image

I changed that because the Dedicated text description is way too long and getting into 3rd line & ruining the alignment.

**Comment by Siddhant-K-code** [comment]
> the link to /cloud doesn't work yet
> when opening the mobile drop-down, "cloud" is already selected? both options should have the same default state

both were intentional & happening because of [this](https://github.com/gitpod-io/website/pull/3816#discussion_r1251942840)

**Comment by chrifro** [comment]
>I changed that because the Dedicated text description is way too long and getting into 3rd line & ruining the alignment. Let's keep it wide for now, we can iterate on it later on.

How come you didn't use the width suggested in [the mockup](https://www.figma.com/file/neSwTwqIeUAtzEIIEAKOPp/%2Fplatform%2Fxyz?type=design&node-id=2%3A4966&mode=design&t=lGuCWXVPNfICoMcs-1)? There it works fine with the line break and it's consistent with all the other drop-downs 
@Siddhant-K-code

**Pull request merged**